content/docs/rosa/prereq-list/index.md

```diff
@@ -0,0 +1,103 @@
+---
+date: '2023-07-27'
+title: Prerequisites Checklist to Deploy ROSA Cluster with STS 
+tags: ["ROSA", "STS"]
+authors:
+  - Byron Miller
+  - Connor Wooley
+  - Diana Sari
+---
+
+## Background
+This is a quick checklist of prerequisites needed to spin up a classic [Red Hat OpenShift Service on AWS (ROSA)](https://developers.redhat.com/products/red-hat-openshift-service-on-aws/overview) cluster with [STS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html). Note that this is a high level checklist and your implementation may vary. 
+
+Before running the installation process, make sure that you deploy this from a machine that has access to:
+- The API services for the cloud to which you provision.
+- Access to `api.openshift.com` and `sso.openshift.com`. 
+- The hosts on the network that you provision.
+- The internet to obtain installation media.
+
+In addition, please refer to the official documentation [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#rosa-aws-prereqs_rosa-sts-aws-prereqs) for more details of the prerequisites in general.
+
+## Accounts and CLIs Prerequisites
+First, let's discuss about the accounts and CLIs you would need to install to deploy the cluster.
+
+### AWS account:
+  - You would need the following details:
+      - AWS IAM User
+      - AWS Access Key ID
+      - AWS Secret Access Key
+  - Ensure that you have the right permissions as detailed [here](https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol.html) and [here](https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html)
+  - Please also refer [here](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#rosa-account_rosa-sts-aws-prereqs) for more details. 
+### AWS CLI (`aws`):
```

Comment:
markdown elements like tables or code  blocks or lists should have a empty line above and below them.

Reply:
theres no tables nor code blocks in there, but i'll give an extra line if that helps

Reply:
yeah sorry I was just giving the examples ... headings should also have a blank line before or after.  basically any kind of markdown element that's not specifically inline.  This comment was meant to apply to the whole doc, just chose this line to pick on.